template.yml

```diff
@@ -0,0 +1,24 @@
+variables:
+  ENDPOINT: https://events.cto.ai/
+
+
+.cto-send-event: &cto-send-event
+    - | 
+       curl --location --request POST "${ENDPOINT}" \
+            --header "`echo 'Authorization: Bearer' ${TOKEN}`" \
+            --header 'Content-Type: application/json' \
+            --header 'x-ops-mechanism: gitlab-job' \
+            --data "{
+              \"repo\":\"${CI_PROJECT_PATH}\",
+              \"branch\":\"${CI_COMMIT_BRANCH}\",
+              \"commit\":\"${CI_COMMIT_SHA}\",
+              \"environment\":\"${CI_ENVIRONMENT_NAME}\",
+              \"event_name\":\"${EVENT_NAME}\",
+              \"event_action\":\"${EVENT_ACTION}\",
+              \"team_id\":\"${TEAM_ID}\",
+              \"meta\":{\"user\":\"${GITLAB_USER_LOGIN}\"}
```

Comment:
Apparently the variable “GITLAB_USER_LOGIN” is only available since GitLab version 10.0, does this not imply specifying some version within the template?

<img width="914" alt="Screen Shot 2023-05-30 at 14 27 01" src="https://github.com/cto-ai/gitlab-template/assets/101744963/4f496832-61af-4824-aea7-b8666fa4e714">

Reply:
Possibly no, as the last version of `10.x` was released almost 5 years ago. Except `CI_COMMIT_BRANCH` all the variables have released by version `10.x`. `CI_COMMIT_BRANCH` got release in `12.6` which got released 3 years ago. 

I may be incorrect but don't think anybody might be using these older version in their private instances.

Reply:
More info about `gitlab` releases https://gitlab.com/gitlab-org/gitlab/-/releases/v12.6.0-ee

Reply:
@nitishkumar71 agree with you, the version 10.0 is from 2017 
https://about.gitlab.com/releases/2017/09/22/gitlab-10-0-released/